bpf/lib/nodeport.h

```diff
@@ -449,6 +450,37 @@ static __always_inline int dsr_reply_icmp6(struct __ctx_buff *ctx,
 		.daddr		= ip6->saddr,
 		.payload_len	= bpf_htons((__u16)payload_len),
 	};
+	struct ipv6hdr inner_ipv6_hdr __align_stack_8 = *ip6;
+	__be16 dport;
+	__s32 l4_dport_offset;
+
+	/*
+	 * DSR changes the destination address from service ip to pod ip and
```

Comment:
nit: please keep the multiline comment format consistent with the rest of the file, e.g.:
```
/* first line
 * second line
 */
```